HIP/hip-410.md

```diff
@@ -109,11 +109,14 @@ section of the transaction (also referred to as callData or inputData) to be
 stored as a Hedera file. In these instances the transaction body contains an
 edited version of the Transaction, replacing a zero length string for the `data`
 field in the RLP set in the `ethereumData` message field and storing the call
```

Comment:
This HIP should also denote the addition of the 32 byte ethereum transaction hash and its externalization in the record.

Reply:
This is derivable data.  Just take the ethTxBytes from the transaction and keccak256 it.  Are we adding that to the TransactionRecord too?

Reply:
So we only need to hash `ethereumData` and not the contents of the callData file bytes? If so, then that should suffice. Perhaps calling this logic out in a mirror node (or record/block) section might be useful. Basically `ethereumNonce` can be extracted from X and `ethereumHash` can be extracted from Y.

Reply:
So the transaction hash will not be easily extractable unless you can marshal the file for the case of large transactions.   If that is too heavy a lift for the mirror node ingest then we can add it to the record, and then the nonce while we are at it.

Reply:
As discussed, I do think we should add `ethereumHash` to the record.